package-lock.json

```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "mattermost-mobile",
-      "version": "2.32.0",
+      "version": "2.33.0",
```

Comment:
Please, remove this from the PR as this will likely cause conflicts

Reply:
Done.